presto-spark-classloader-interface/src/main/java/com/facebook/presto/spark/classloader_interface/PrestoSparkConfiguration.java

```diff
@@ -37,7 +40,7 @@ public PrestoSparkConfiguration(
             Map<String, String> configProperties,
             String pluginsDirectoryPath,
             Map<String, Map<String, String>> catalogProperties,
-            String metadataStorageType,
+            Map<String, String> prestoSparkProperties,
```

Comment:
`prestoSparkProperties` sounds like a very generic term here. Also, the term `properties` is confusing. We are using this map to pass some setup information. Should we use something like `prestoSparkExecutionSetupOptions`?